src/compiler/types.ts

```diff
@@ -4010,7 +4010,8 @@ namespace ts {
         Tsx = ".tsx",
         Dts = ".d.ts",
         Js = ".js",
-        Jsx = ".jsx"
+        Jsx = ".jsx",
+        Json = ".json"

```

Comment:
There are some places that currently use the string literal `".json"` that could use this instead.